fs/operations/rc_test.go

```diff
@@ -306,6 +306,61 @@ func TestRcStat(t *testing.T) {
 	})
 }
 
+// operations/settier: Set the storage tier of a fs
+func TestRcSetTier(t *testing.T) {
+	ctx := context.Background()
+	r, call := rcNewRun(t, "operations/settier")
+	file1 := r.WriteObject(context.Background(), "file1", "file1 contents", t1)
+	r.CheckRemoteItems(t, file1)
+
+	if !r.Fremote.Features().SetTier {
```

Comment:
If you put the Skip above the WriteObject then the tests will do a bit less work on unsupported backends.

Reply:
Updating this now